uk.co.compucorp.civicrm.hrleaveandabsences/views/absence-tab/components/absence-tab-entitlements.html

```diff
@@ -1 +1,154 @@
-Entitlements
+<section class="panel panel-default panel-default-plain">
+  <header class="panel-heading">
+    <h2 class="panel-title">Contract Entitlements</h2>
+    <hr/>
+  </header>
+  <div class="table-responsive">
```

Comment:
Ok yes you should keep it, but `table-responsive` class should go to `table` tag itself, and we dont need the div then